services/frontend/src/store/index.js

```diff
@@ -9,56 +10,116 @@ Vue.use(Vuex)
 // dispatch is for actions, actions are asynchronous
 // don't know much about modules
 
-export default new Vuex.Store({
-  state: {
-    currentPatient: {
-      patientIdType: 'pesel',
-      patientId: null,
-      firstName: null,
-      secondName: null,
-      surname: null,
-    },
-    analysis: {
-      file: null,
-      result: null
-    },
-    patients: []
+const state = {
+  currentPatient: {
+    patientIdType: 'pesel',
+    patientId: null,
+    firstName: null,
+    secondName: null,
+    surname: null,
   },
-  mutations: {
-    setPatient(state, patient) {
-      state.currentPatient = patient;
-    },
-    setAnalysisFile(state, file) {
-      state.analysis.file = file
-    },
-    setAnalysisResult(state, result) {
-      state.analysis.result = result
-    },
-    setPatients(state, patients) {
-      state.patients = patients
-    }
+  analysis: {
+    recordingID: null,
+    analysisID: null,
+    file: null,
+    result: null,
+  },
+  patients: []
+}
+
+export const mutations = {
```

Comment:
I am not sure if it is a good name, why not just setters? You have gettters below

Reply:
this name comes from vuex, and is standat vuex naming. not sure we can even change it